src/components/SearchFacet/SearchFacet.tsx

```diff
@@ -133,9 +143,23 @@ export const SearchFacet = (props: ISearchFacetProps): ReactElement => {
     padding: isDragging ? '4px' : undefined,
   };
 
+  // keyboard focus search facet and item
+  const [keyboardFocus, setKeyboardFocus] = useState<KeyboardFocusItem>({ facetId: null, index: [] });
```

Comment:
I'm thinking that this is probably defeating the purpose of the context you create below. Since this is causing the whole tree to get rerendered when we change focus.  

https://github.com/adsabs/nectar/assets/6970899/7ae5554e-68aa-4cad-973b-bb65417aed54

Probably for these list sizes it's not making a huge difference, however.  I think if wanna be efficient about it, this needs to be moved down to the item level and have the component watch the store for changes in focus state